tests/api/v1/datadog/features/ip_ranges.feature

```diff
@@ -0,0 +1,11 @@
+@v1 @endpoint(ip-ranges)
+Feature: IP Ranges
+  Get a list of IP prefixes belonging to Datadog.
+
+  Background:
+    Given an instance of "IPRanges" API
+
+  Scenario: List IP Ranges
+    Given new "GetIPRanges" request
+    When the request is sent
+    Then the status is 200 List of IP ranges.
```

Comment:
This one reads a little weird to me. Maybe drop `List of IP ranges` or "Then the status from List of IP ranges is 200"

Reply:
`List of IP ranges` comes from response description. It's generated and not used but it should improve readability if used correctly.